libs/newsletter/guides/Advanced/customizing-submission-data.md

```diff
@@ -0,0 +1,60 @@
+# Customizing Submission Data
+Whenever you are setting up your newsletter subscription within your application, you must decide what information is important for your user to include for the subscription. This guide will walk you through how to include all the desired information when the user submits their subscription using the `DaffNewsletter` library.
```

Comment:
> This guide will walk you through how to include all the desired information when the user submits their subscription using the `DaffNewsletter` library.

This guide will walk you through including your desired newsletter payload when a user submits their newsletter subscription using the DaffNewsletter library.